### PR TITLE
[Merged by Bors] - Added docs for ``.apply()``in basic usage of ``systemState``

### LINKED ISSUE #5940: Docs for SystemState should mention using apply() to effect Commands
https://docs.rs/bevy/latest/bevy/ecs/system/struct.SystemState.html

This detail pulled from the Bevy Cheat Book should appear in the Basic Usage section for SystemState:

> If you are using Commands, you can choose when you want to apply them to the world. You need to manually call .apply(world) on the SystemState, to apply them.

### PR DESCRIPTION
**Pull request by zeroacez**
# Objective

Fixes #5940 

## Solution

Added the suggested comment.

### CODE REVIEW

File: crates/bevy_ecs/src/system/function_system.rs [hunk 1]
```diff
@@ -106,6 +106,9 @@ impl SystemMeta {
 /// // Use system_state.get_mut(&mut world) and unpack your system parameters into variables!
 /// // system_state.get(&world) provides read-only versions of your system parameters instead.
 /// let (event_writer, maybe_resource, query) = system_state.get_mut(&mut world);
+///
+/// // If you are using Commands, you can choose when you want to apply them to the world.
+/// // You need to manually call .apply(world) on the SystemState, to apply them.
```
Comment (alice-i-cecile) [inline crates/bevy_ecs/src/system/function_system.rs:111]:
```suggestion
/// // You need to manually call `.apply(world)` on the [`SystemState`] to apply them.
```

File: crates/bevy_ecs/src/system/function_system.rs [hunk 2]
```diff
@@ -106,6 +106,9 @@ impl SystemMeta {
 /// // Use system_state.get_mut(&mut world) and unpack your system parameters into variables!
 /// // system_state.get(&world) provides read-only versions of your system parameters instead.
 /// let (event_writer, maybe_resource, query) = system_state.get_mut(&mut world);
+///
+/// // If you are using Commands, you can choose when you want to apply them to the world.
```
Comment (alice-i-cecile) [inline crates/bevy_ecs/src/system/function_system.rs:110]:
```suggestion
/// // If you are using [`Commands`], you can choose when you want to apply them to the world.
```

**Review by alice-i-cecile — CHANGES_REQUESTED** [review]
Once those links are added and working this LGTM :)

**Review by james7132 — APPROVED** [review]
LGTM sans Alice's comments.

**Comment by alice-i-cecile** [comment]
bors retry

**Pull request closed without merging**